packages/react/src/components/Header/Header.test.jsx

```diff
@@ -285,4 +286,99 @@ describe('Header', () => {
     fireEvent.click(menuItem);
     expect(screen.getByLabelText('user')).toBeTruthy();
   });
+
+  it('should move icons into overflow menu when area too small', () => {
+    Object.defineProperty(window, 'innerWidth', { writable: true, configurable: true, value: 375 });
```

Comment:
Should we restore this once done testing?

Reply:
good eye.